Whole site

jpdf00 opened the pull request:
This is the first project in Microverse's HTML and CSS module. It consists of a reproduction of a page in the New York Times website. It displays the text, images and video of the original as well as some of the links. Advertisements and many of the interactive elements have been left out.

Built using HTML and CSS only.

jpdf00 committed:
Created linters.yml

Created folders and linters file for code control on pull requests.

jpdf00 committed:
Created index.html

Created the html file to write te site's code.

jpdf00 committed:
Created style.css

Create css file to style the webpage in the repository.

jpdf00 committed:
Updated index.html

Added the head and a bare skeleton for the elements that are going to be used.

jpdf00 committed:
Update to index.html

Added new elements to the skeleton.

Commit:
Added to the html skeleton several elements

jpdf00 committed:
Update index.html

Starting to populate site with information.

jpdf00 committed:
Update index.html

Increased article, add links.

jpdf00 committed:
Update index.html

Add all paragraphs.
Add Video.

Commit:
Added buttons and images to all navbars

jpdf00 committed:
Update index.html and style.css

Add color and font to figure captions.
Oraganization improvment.

Commit:
re-sized icons, positioned nav-bars

jpdf00 committed:
Update index.html and style.css

Add section for central image.
Add style for buttons.

Commit:
Separated elements in the nva bar, left and right justified

jpdf00 committed:
Update Style.css

Adjusts to the flexbox model.

Commit:
Added footer links

jpdf00 committed:
Update style.css and index.html

Fixed several issues regarding positioning.

Fixed top nav bar.

Commit:
Positioned footer elements with flexbox

jpdf00 committed:
Update index.html and style.css

Finished the footer.

jpdf00 committed:
Update index.html and style.css

Styled the aside.

jpdf00 committed:
Update style.css and index.html

Finished positioning

jpdf00 committed:
Update index.html

Correct indentation.

Commit:
Completed README file

Commit:
fixed issues pointed out by linter

jpdf00 committed:
Fixed issues.

Fixed linter erros.

jpdf00 committed:
Fixed issues.

Fixed Linter errors.

ElijahTobs reviewed (changes requested):
# Status: Changes Required :recycle: 

## Hi @jpdf00 and @German-Cobian 
Welcome to Microverse :tada: and congratulations on submitting your first project.
Your projects do not meet the requirements, as such, you will have to make some changes and submit for another review

**Required Changes are mandatory changes you need to make before your project can be approved while Optional changes are nice-to-haves or recommendations which would not stop your projects from getting approved.**

## [REQUIRED]
- [x] You should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Add a little margin to the top of your navbar.
- [x] Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). 
- [x] The Editors' Pick is not well aligned to the right. You should also set a width/max-width to it.
- [x] In **Theory of Inflation section**, the header and the following paragraph are to be aligned to the left, not the center.
- [x] Make sure you add the **suggested newsletter section**
![image](https://user-images.githubusercontent.com/16058470/100301506-92d68980-2f98-11eb-9785-4dd748dd6b31.png)
- [x] The image in the **Spirals in the sky** section is not of the same width and height as that of the original page.
- [x] In the footer, the section with a class of linked-list should be wider. Increase the width of that section or that of the entire footer.
- [x] There should be 2 horizontal lines right above your footer.
- [x] Add enough margin before your footer.
- [x] In your footer, the first 4 items under the subscribe section are missing the icons. Please add them.
- [x] You should also add some margin before and after the section with a class of **links-ribbon.**

## [OPTIONAL]

- [ ] I noticed that some of the links on your page actually re-direct to the main site. You can just leave the **href empty or add #**.

### Make those changes and submit for another code review.
### Please, make sure you submit the same pull request link. Do not create a new pull request!

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

Commit:
began the suggested links section of the article

jpdf00 committed:
Update index.html

Add imagens to Section "newsletter-child suggested".

jpdf00 committed:
Update index.html

Finish section More and Most Popular.

jpdf00 committed:
Fixed Linters

Change figcaption for p.

jpdf00 committed:
Update index.html

Fix the use of figure and figcaption elements.

Commit:
floated editor's picks sidebar and centered elements after eliminating redundant flexbox instances

Commit:
cleaned up the aside element

jpdf00 committed:
Update index.html

Create Newsletter section.

Temmarie reviewed (changes requested):
### Changes Required! :recycle:
Hi Team ! 👋 👋 

### Welcome to Microverse once again! You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them as well as the new ones, and submit them for another code review.

**General**

Please fix the linter issues. Do reach out to me incase you have issues with it.

**Recommended**

- [x] Fix the navbar, you currently have the buttons merging together. Please add some spacings in between.

- [x] You have a horizontal overflow. Please fix this.

- [x] You currently have the "Editors' picks" at a position where it should not be. Please refer to the actual site for clarity.

**Original**

![Screenshot (54)](https://user-images.githubusercontent.com/49621279/100397028-fcfa3780-3047-11eb-9ba6-8631f2cb8fe1.png)

**Yours**

![Screenshot (55)](https://user-images.githubusercontent.com/49621279/100397037-0d121700-3048-11eb-8c3d-9f1baa4ed14d.png)

- [x] The images in the "More in space and astronomy" section are lacking some gaps.

**Original**

![Screenshot (58)](https://user-images.githubusercontent.com/49621279/100397099-5d897480-3048-11eb-9a38-20827eeeafc2.png)

**Yours**

![Screenshot (57)](https://user-images.githubusercontent.com/49621279/100397107-6417ec00-3048-11eb-8068-cd02d4a78d8a.png)

- [x]  Please fix the footer as well for it to look as similar as possible to the actual sites.

**Original**

![Screenshot (48)](https://user-images.githubusercontent.com/49621279/100397168-94f82100-3048-11eb-87f2-38b7a0f275f9.png)

**Yours**

![Screenshot (59)](https://user-images.githubusercontent.com/49621279/100397152-89a4f580-3048-11eb-9eaf-d8eef7b65d18.png)


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

jpdf00 committed:
Update index.html and style.css

Fix linters.

jpdf00 committed:
Update style.css

Fix linters.

jpdf00 committed:
Update index.html and style.css

Fixed editor's picks section

Commit:
fixed format problems

jpdf00 committed:
Update style.css

Fix linters.

jpdf00 committed:
Update style.css

Fix linters.

jpdf00 committed:
Update README.md

Use Microverse Template

ebeagusamuel reviewed (approved):
# Project Approved :100: 
Hello @jpdf00 and @German-Cobian :wave: 
Nice work on this project. You can now go ahead and merge this branch.
Optionally you can try to improve on the following
- [x] The screenshot in your readme is not loading
- [x] Delete files and folders that you are not making use of from the project, e.g DS_Store.
- [ ] Fix horizontal scroll.

Wish you good luck in the next project.
slack(@ebeagusamuel)

jpdf00 committed:
Update README.md and Organize Folder

Add social media links on README.md.
Add file project-01-screenshot.png and linked it to README.md.
Remove unecessary files.

jpdf00 committed:
Update README.md

Fix Screenshot.

jpdf00 committed:
Update README.md

Fix Screenshot
